packages/edit-site/src/components/dataviews/view-actions.js

```diff
@@ -183,40 +207,10 @@ function SortMenu( { dataView } ) {
 						}
 						side="left"
 					>
-						{ Object.entries( sortingItemsInfo ).map(
```

Comment:
Is the change in this file necessary. Asking because I'm updating this code in #55294
What was changed exactly here?

Reply:
I'm guessing you just want to reuse the "submenu" for a field in the other PR. I think we need to choose between:

 - Not reusing this component and just duplicating it because one can be independent from tanstack but the other should not.
 - Reusing the one that is not dependent on tanstack. 

But since the parent components of both have different requirements, I might think that it's better to not reuse here.

Reply:
I'm just reusing the sort option menu items that are identical in terms of UI. I get the refactoring in your PR, but I don't get how they are different.  The both depend on the view config. Maybe the could live in a different file? 🤔

Reply:
The reusable thing is menu items but the whole menu is different which makes me think it's a bit weird to reuse but it's fine. I can handle that. I changed the props of the component (not use dataView object)

Reply:
I mean, it's an easy update to duplicate or reuse and change approach later on. I have no strong opinions. Maybe we should land your PR first and see then. Though I don't think it makes much difference at this point. What do you think?

Reply:
Yeah, I'm not too concerned either, whatever lands first is fine :)